tests/test_utilities.py

```diff
@@ -1540,6 +1540,36 @@ def test_is_course_run_about_to_end(self, course_run, expected_boolean):
         assert utils.is_course_run_about_to_end(course_run) == expected_boolean
 
     @ddt.data(
+        (
+            # Test with two Current runs, but one not published. Should choose published one.
+            {
+                "course_runs": [
+                    fake_catalog_api.create_course_run_dict(
+                        availability="Current",
+                        start="2015-10-15T13:11:03Z",
+                        status="unpublished",
+                        end="2014-10-15T13:11:03Z",
+                        enrollment_start=None,
+                        enrollment_end=None,
+                    ),
+                    fake_catalog_api.create_course_run_dict(
```

Comment:
this is not an exact but close approx. of the case at https://discovery-internal.edx.org/api/v1/courses/NYIF+CR.3x/